### ISSUE TITLE
Add an example of use

### ISSUE THREAD

**Issue by fgriberi** [headless-ninja/hn-react-webform#7]
Please, could it be possible to add an use example?

**Comment by carolpettirossi** [comment]
Hi, 

I'm also interested in how to use this. To give some context, my scenario would beth exposing the webform via graphql using the graphql_webform module. 
And then render it using this React Component. Is that possible?
If not, what the component expects? 

Thanks

**Comment by K3vinF** [comment]
Curious as well!
How is the webform data made available to React? It doesn't look to be using webform_jsonschema
Does it work with the the webform_rest module? Or is there an another (custom?) solution to export the webform data?

**Comment by nnmlss** [comment]
Sorry for the offtopic, but without using this component I was able to create a custom react module, which reads a webform submissions via api made with views module. A view of fields by type Serializer : json, authentificated by coockie, and I am able to edit the webform submission results using webform_rest and restui, again with coockie authentification and axios  into react side.  Hope this answers your question.

**Comment by K3vinF** [comment]
That is good to know.
We are currently using the webform_jsonschema module via a (custom) graphQl implementation, and the webform_jsonschema rest end point. This also works. However I notice that not all of the webform options / layouts are translated to the webform_jsonschema.
In that light I was / am curious how this solution compares.

**Comment by nnmlss** [comment]
Yup, this was the main reason to subscribe to this issue but it looks like the dev has no time to answer. Good luck
